demo/x-app.html

```diff
@@ -16,19 +16,23 @@
 
     :host {
       @apply(--layout-horizontal);
+      @apply(--paper-font-body2);
     }
 
     [nav] {
-      width: 300px;
       @apply(--layout-vertical);
-      background-color: #ddd;
+
+      width: 300px;
```

Comment:
Does this look ok on mobile? maybe you want `max-width` (I have learnt this the hard way)